temba/msgs/tests.py

```diff
@@ -2103,3 +2107,120 @@ def test_render(self):
         # exception if tag not used correctly
         self.assertRaises(ValueError, self.render_template, '{% load sms %}{% render with bob %}{% endrender %}')
         self.assertRaises(ValueError, self.render_template, '{% load sms %}{% render as %}{% endrender %}')
+
+
+class CeleryTaskTest(TransactionTestCase):
```

Comment:
Hrmm.. any chance you could use insane Python multiple inheritance to get the best of both? Looking at `TransactionTestCase` I don't think its magic overlaps much with what is provided by TembaTest so I don't think they would walk over each other.

Sorry to put this back in your court but man our tests are already so full of repetition already we are trying not to have that debt grow even more.